mpt-witness-generator/witness/leaf.go

```diff
@@ -474,21 +476,23 @@ func prepareStorageLeafInfo(row []byte, valueIsZero, isPlaceholder bool) ([]byte
 			// [227,48,161,160,187,239,170,18,88,1,56,188,38,60,149,117,120,38,223,78,36,235,129,201,170,170,170,170,170,170,170,170,170,170,170,170]
 			key[0] = row[0]
 			key[1] = row[1]
+			keyRlpLen = 1
```

Comment:
You can put the following in between the else L471 and the if line 472 and remove from the next lines:
```
keyRlpLen = 1
keyRlp = make([]uint8, keyRlpLen)
copy(keyRlp, row[:keyRlpLen])
valueRlpLen = 1
```